frontend/src/metabase/dashboard/components/DashboardTabs/DashboardTabs.unit.spec.tsx

```diff
@@ -285,6 +286,19 @@ describe("DashboardTabs", () => {
 
         expect(queryTab(newName)).toBeInTheDocument();
       });
+
+      it("should allow renaming via double click", async () => {
+        setup();
+        const newName = "Another cool new name";
+        const inputWrapperEl = screen.getAllByTestId(INPUT_WRAPPER_TEST_ID)[0];
```

Comment:
We have to double click the wrapper element instead of the input itself since the input will not register a click event as it as the `disabled` attribute on it.